modules/apps/calendar/calendar-web/src/main/resources/META-INF/resources/js/legacy/calendar_util.js

```diff
@@ -123,7 +123,18 @@ AUI.add(
 			fillURLParameters(url, data) {
 				url = Lang.sub(url, data);
 
-				return url.replace(REGEX_UNFILLED_PARAMETER, '');
+				url = url.replace(REGEX_UNFILLED_PARAMETER, '');
```

Comment:
hey, @carolmariaabb, we can avoid this line by doing all necessary manipulations of the url string on the same line. like this: url = Lang.sub(url, data).replace(REGEX_UNFILLED_PARAMETER, '');

Additionally, we could store that value in a const instead of reassigning the parameter itself to improve the code readability. This is just a suggestion since the code was already like this and there are other occurrences throughout this file, so feel free to do as you see fit!